### PR TITLE
Migrate go mod from dep

### PR DESCRIPTION
**Pull request by b4b4r07**
## WHAT

Use go mod, instead of dep.

## WHY

Go modules can be available from [1.13](https://golang.org/doc/go1.13).

### CODE REVIEW

**Comment by b4b4r07** [comment]
CI failing...

**Comment by b4b4r07** [comment]
Fixed

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,20 @@
+module github.com/mercari/tfnotify
+
+go 1.12
```
Comment (toshi0607) [inline go.mod]: Is 1.13 preferable?

File: .circleci/config.yml [hunk 2]
```diff
@@ -7,14 +7,13 @@ jobs:
   build:
     <<: *defaults
     docker:
-      - image: golang:1.10-stretch
+      - image: golang:1.13
```
Comment (toshi0607) [inline .circleci/config.yml]: Did you remove `-stretch` intentionally? `1.13-stretch` image seems to be smaller than `golang:1.13`
```
golang                      1.13-stretch        b130f0a1c39b        2 weeks ago         763MB
golang                      1.13                272e3f68338f        2 weeks ago         803MB
```

Comment (b4b4r07) [reply .circleci/config.yml]: !

thanks! I'm gonna update it. Also I'll migrate to GitHub action from circleci!

**Pull request merged**